src/systems/media-frames.js

```diff
@@ -262,22 +332,38 @@ AFRAME.registerComponent("media-frame", {
 
   capture(capturableEntity) {
     if (NAF.utils.isMine(this.el) || NAF.utils.takeOwnership(this.el)) {
-      this.el.setAttribute("media-frame", {
-        targetId: capturableEntity.id,
-        originalTargetScale: new THREE.Vector3().copy(capturableEntity.object3D.scale)
-      });
-      const worldPosition = new THREE.Vector3();
-      this.el.object3D.getWorldPosition(worldPosition);
-      capturableEntity.object3D.position.copy(worldPosition);
-      const worldQuat = new THREE.Quaternion();
-      this.el.object3D.updateWorldMatrix(true);
-      this.el.object3D.getWorldQuaternion(worldQuat);
-      capturableEntity.object3D.setRotationFromQuaternion(worldQuat);
-      capturableEntity.object3D.scale.setScalar(
-        scaleForAspectFit(this.data.bounds, capturableEntity.getObject3D("mesh").scale)
-      );
-      capturableEntity.object3D.matrixNeedsUpdate = true;
-      capturableEntity.components["floaty-object"].setLocked(true);
+      const update = () => {
+        this.el.setAttribute("media-frame", {
+          targetId: capturableEntity.id,
+          originalTargetScale: new THREE.Vector3().copy(capturableEntity.object3D.scale)
+        });
+
+        capturableEntity.object3D.scale.set(1, 1, 1);
+        capturableEntity.object3D.quaternion.set(0, 0, 0, 1);
+        capturableEntity.object3D.updateMatrixWorld();
```

Comment:
Don't think this is needed but you should set `matrixNeedsUpdate` since scale/quaternion was just changed.